mono/metadata/loader.c

```diff
@@ -1768,7 +1768,8 @@ mono_get_method_checked (MonoImage *image, guint32 token, MonoClass *klass, Mono
 
 	/* We do everything inside the lock to prevent creation races */
 
-	error_init (error);
+	error_init (error); // for the assert
```

Comment:
Same as https://github.com/mono/mono/pull/6406/files#diff-f085caaf39e861140ff40b9ef0c516d9R1028

Reply:
I think I changed it back to error_init, or else I have error_init_check available -- "check" being assert or warn, depending on a global.